process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -48,40 +48,40 @@ type storageBootstrapper struct {
 
 func (st *storageBootstrapper) loadBlocks() error {
 	var err error
-	var headerInfo bootstrapStorage.BootstrapData
+	var lastHeader bootstrapStorage.BootstrapData
```

Comment:
Here is ok to keep headerInfo as variable name, so I would changed it back from lastheader to headerInfo. I was referred to change the name of the variable from BootstrapData structure from HeaderInfo to LastHeader

Reply:
done.